app/views/admin/link_check_reports/_link_check_report.html.erb

```diff
@@ -15,11 +15,25 @@
     </section>
   <% elsif report.broken_links.present? %>
     <section class="alert alert-warning">
-      <h3 class="remove-top-margin">Broken links</h3>
-      <p>We’ve found links in this document that may be broken:</p>
-      <ul>
-        <% report.broken_links.each do |link| %>
-          <li><%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'link-inherit' %></li>
+      <ul class="issue-list">
+        <% report.links.sort_by(&:status).group_by(&:status).each do |status, links| %>
+          <% next unless %w(broken caution).include? status %>
+          <li>
+            <h3 class="remove-top-margin"><%= t "broken_links.#{status}.title" %></h3>
+            <p><%= t "broken_links.#{status}.subheading" %></p>
+          </li>
+          <% links.each do |link| %>
+            <li>
+              <%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'link-inherit' %>
+              <details>
+                <summary class="display-issue-details">See more details about this link</summary>
+                <p class="issue-summary"><%= link.problem_summary %></p>
+                <% if link.suggested_fix %>
+                  <p class="issue-summary">Suggested fix: <%= link.suggested_fix %></p>
+                <% end %>
+              </details>
+            </li>
+          <% end %>
```

Comment:
This HTML feels a little strange where we have headers inside the list.

I feel like this would feel more semantic were you to have it like:
```
<h3 class="remove-top-margin"><%= t "broken_links.broken.title" %></h3>
<p><%= t "broken_links.broken.subheading" %></p>
<ul class="issue-list">
 ....
</ul>
```
and then just have the issues in the list, as that is really the part of this that is a list. What do you think?